### PR TITLE
feat: [CCH-50] modular dashboard navbar

### PR DESCRIPTION
**Pull request by JudeDavis1**
Modular navbar and layouts to define which navbar to show based on route.

### CODE REVIEW

File: packages/frontend/app/home/components/home.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import FeatureItem from "@/components/feature-item";
+import TextAnim from "@/components/text-animation";
+
+export function Home() {
+  // For now just using images from a random watchtower
+  const features = [
+    {
+      label: "Display media for the meeting",
+      img_url:
```
Comment (j-koziel) [inline packages/frontend/app/home/components/home.tsx]: this might be a bit of a nit pick but maybe use camelCase here for consistent naming 👍

**Review by j-koziel — CHANGES_REQUESTED** [review]
Just one small change... nothing major

File: packages/frontend/app/home/components/home.tsx (hunk 1 above)
Comment (JudeDavis1) [reply packages/frontend/app/home/components/home.tsx]: Oh loool I didn't write that. Wasn't that you? 😜 I'll fix it when I can

**Review by j-koziel — APPROVED** [review]
LGTM totally forgot about this one 😬

**Pull request merged**